client/Overview/FeaturedProduct.jsx

```diff
@@ -1,48 +1,113 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import FeaturedReviewLink from './FeaturedReviewLink.jsx';
+import Stars from '../SharedComponents/Stars.jsx';
+// import FeaturedReviewLink from './FeaturedReviewLink.jsx';
 
 class FeaturedProduct extends React.Component {
   // selectProductInfo={selectProductInfo} viaOvC
   constructor() {
     super();
     this.state = {};
+    // this.ScrollToReviewSection = this.ScrollToReviewSection.bind(this);
   }
 
+  // ScrollToReviewSection() {
+  //   document.getElementByClassName('ratings-and-reviews-container').scrollIntoView(true)
```

Comment:
getElementsByClassName returns an array; if this isn't working, I'd suggest trying:

`document.getElementsByClassName('ratings-and-reviews-container')[0].scrollIntoView(true);`

And, if you want to make the scroll smooth, try:

`document.getElementsByClassName('ratings-and-reviews-container')[0].scrollIntoView({behavior: "smooth", block: "start", inline: "nearest"})`

(Don't forget to make getElement**s**ByClassName plural!